Pages/input.html

```diff
@@ -154,23 +156,24 @@ <h3 class="my-2">Result</h3>
             <hr />
             <h3 class="my-2">Available variations</h3>
             <p>
```

Comment:
are you nesting ol in the p tag for any specific reason? If no then you can take the ol out of the p tag